### PR TITLE
[Spanish] Translate guides configuration glossary

### PR DESCRIPTION
**Pull request by kinxos**
Pages translated to Spanish for #543

📝 guides -> configuration-glossary

- [x] configuration-dev.md
- [x] configuration-dir.md
- [x] configuration-modern.md

### CODE REVIEW

**Comment by debs-obrien** [comment]
@Dawntraoz @alvarosaburido can either of you give this a quick review please

**Comment by kinxos** [comment]
@Dawntraoz @alvarosaburido There's two words particularly that I didn't know how to translate, so I didn't: build (as in 'the build process') and legacy (as in 'legacy bundle').

**Comment by kinxos** [comment]
Texts have been updated, with italized non-translatable words.

File: content/es/guides/configuration-glossary/configuration-modern.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+| Modo      | Modo moderno |
+| --------- | :---------: |
+| universal |   server    |
+| spa       |   client    |
+
+- El modo moderno para `nuxt generate` sólo puede ser `client`
+- Utiliza [`render.crossorigin`](/guides/configuration-glossary/configuration-render#crossorigin) para definir el atributo `crossorigin` en `<link>` y `<script>`
+
+> Por favor, visita el [excelente post de Phillip Walton](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/) para más información sobre builds modernos.
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-modern.md:48]: Por favor, visita este [excelente post de Phillip Walton](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/) para más información sobre builds modernos.

Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-modern.md]: Hecho!

**Comment by debs-obrien** [comment]
great work guys. 😃

**Pull request merged**